### DISCUSSION TITLE
Help with the wiring for the mic

### DISCUSSION THREAD

**Discussion by jwhengeveld** [justLV/onju-voice#9] [Q&A] [answered]
First of all -> Great project! I have been busy tinkering with the ESP32 and the recommended components but i am struggling with the correct wiring for the mic

The way i have set it up now: BOARD_NAME "V3"

**MIC** / **SPH0645** 
SEL -> GROUND
LRCL -> I2S_WS_PIN (13) 
DOUT-> I2S_IN (17)
BCLK -> I2S_BCK_PIN (18)
GND -> GROUND
3V -> 3V3

**AMP/SPEAKER** / **MAX98357A**
LRC -> SPEAKER_EN (21)
BCLK -> I2S_BCK_PIN (18)
DIN -> I2S_OUT (12)
GAIN -> x
SD -> 3V3
GND -> GROUND
VIN -> 5V 

Do you see what i am doing wrong in the wiring?

![ESP32-S3-DEV-KIT-N8R8-details-13](https://github.com/justLV/onju-voice/assets/690293/3bb6525c-146f-4d31-bbf8-dd7f80c73368)

**Comment by justLV** [comment]
LRC is likely Left/Right clock which is the same as Word Select (WS) so it probably needs to be connected to both the mic & speaker.

SPEAKER_EN doesn't look like it needs to connect to the MAX9... at all

**Reply by jwhengeveld** [reply]
Thank you for your reply! So I don't need to connect any wire to SPEAKER_EN? The speaker does work now with the wiring, the problem is the mic that does not record any sounds, only noise

So this should be the wiring? (Including where the mute is connected?). What was your wiring in your esp32 dev board? I used the same components 

MIC / SPH0645
SEL -> MUTE (38)
LRCL -> I2S_WS_PIN (13)
DOUT-> I2S_IN (17)
BCLK -> I2S_BCK_PIN (18)
GND -> GROUND
3V -> 3V3

**Reply by jwhengeveld** [reply, marked as answer]
I have resolved it. SEL had to be connected to the 3V3. I do not know where the MUTE pin (38 in V3 board) should be connected to
